tests/test_types_basic.py

```diff
@@ -471,36 +471,14 @@ def getquoted(self):
         self.assertEqual(ext.adapt(foo((1, 2, 3))).getquoted(), 'bar')
 
 
-@decorate_all_tests
-def skip_if_cant_cast(f):
-    @wraps(f)
-    def skip_if_cant_cast_(self, *args, **kwargs):
-        if self._cast is None:
-            return self.skipTest("can't test bytea parser: %s - %s"
-                % (self._exc.__class__.__name__, self._exc))
-
-        return f(self, *args, **kwargs)
-
-    return skip_if_cant_cast_
-
-
-@skip_if_cant_cast
+@skip_if_windows
```

Comment:
As you taught us the arts of using `unittest.skipIf()` :grin: I think this would be better expressed with an:

    unittest.skipIf(
        platform.system() == 'Windows',
        "Not testing because we are useless with ctypes on Windows")

or something like that? `skip_if_windows` sounds like there is some feature which is not available on win, whereas this is only a test shortcoming

Reply:
Done! :slightly_smiling_face: